exchanges/deribit/deribit_websocket.go

```diff
@@ -827,23 +831,23 @@ func (d *Deribit) handleSubscription(method string, subs subscription.List) erro
 		subAck[c] = true
 	}
 	if len(subAck) != len(subs) {
-		err = common.ErrUnknownError
+		err = errSubscriptionFailureDetected
 	}
 	for _, s := range subs {
 		if _, ok := subAck[s.QualifiedChannel]; ok {
-			err = common.AppendError(err, d.Websocket.AddSuccessfulSubscriptions(d.Websocket.Conn, s))
+			if strings.Contains(method, "subscribe") {
```

Comment:
I haven't tested yet, but "unsubscribe" contains the string "subscribe" and so both would likely fall in here

Reply:
[pushed](https://github.com/thrasher-corp/gocryptotrader/pull/1717/commits/7401285cad333b21b7295baf48c1c85b139c621a)